app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :comment, presence: true
+  validates :user_id, presence: true
+  validates :post_id, presence: true
+end
```

Comment:
- [ ] You are trying to validate a presence for a column that does not exist on your table, `:comment`. We need to change that for `:body` to make it possible to create a comment.

- [ ] There is no need to validate foreign keys. They are required by default.